app/templates/index.html

```diff
@@ -104,7 +104,7 @@ <h2 id="supplier-message-heading">Sell services</h2>
           {% if logged_in %}
             <p>
               <a href="/suppliers" class="top-level-link">
-                <span>View your services and account details</span>
+                View your services and account details
```

Comment:
I think the `{% if logged_in %}` condition here needs to be more specific - it should only show this link/sidebar if logged in as a supplier, not as a buyer.